docs/overview/quickstart.mdx

```diff
@@ -40,52 +40,68 @@ TestDriver isn't just a computer-use agent, it's an entire AI-native workflow fo
     </Tabs>
 
   </Step>
-  <Step title="Create your first test">
-    Create a new file called `testdriver/test.yaml`, and paste the following [YAML](/commands) code into it.
+  <Step title="Run an example test">
+      
+    Download the TestDriver GitHub repository and run the web example test.
 
-    ```yaml testdriver/test.yaml
-    steps:
-      - prompt: Navigate to https://app.testdriver.ai
-      - prompt: Verify the login form is displayed
+    ```bash
+    git clone https://github.com/testdriverai/cli testdriverai
```

Comment:
We can also recommend people just clone the examples with something like https://github.com/Rich-Harris/degit

```shell
npx degit testdriverai/testdriverai/examples/web
```